name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/fintechgroupbank/FinTechGroupBankPDFExtractorTest.java

```diff
@@ -2262,6 +2330,60 @@ public void testFinTechWertpapierAusgang06()
                         is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(0.00))));
     }
 
+    @Test
+    public void testFinTechWertpapierAusgang07()
+    {
+        FinTechGroupBankPDFExtractor extractor = new FinTechGroupBankPDFExtractor(new Client());
+
+        List<Exception> errors = new ArrayList<>();
+
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "FinTechWertpapierAusgang07.txt"),
+                        errors);
+
+        assertThat(errors, empty());
+        assertThat(countSecurities(results), is(2L));
+        assertThat(countBuySell(results), is(2L));
+        assertThat(countAccountTransactions(results), is(1L));
+        assertThat(results.size(), is(5));
+        new AssertImportActions().check(results, CurrencyUnit.EUR);
+
+        // check security
+        assertThat(results, hasItem(security( //
+                        hasIsin("DE000CM35Z36"), hasWkn(null), hasTicker(null), //
+                        hasName("COMMERZBANK INLINE09EO/SF"), //
+                        hasCurrencyCode("EUR"))));
+
+        // check buy sell transaction
+        assertThat(results, hasItem(sale( //
+                        hasDate("2009-12-29"), hasShares(650.00), //
+                        hasSource("FinTechWertpapierAusgang07.txt"), //
+                        hasNote("Transaktion-Nr.: 203036888"), //
+                        hasAmount("EUR", 0.65), hasGrossValue("EUR", 0.65), //
+                        hasTaxes("EUR", 0.00), hasFees("EUR", 0.00))));
+
+        // check tax refund transaction
+        assertThat(results, hasItem(taxRefund( //
+                        hasDate("2009-12-29"), hasShares(650.00), //
+                        hasSource("FinTechWertpapierAusgang07.txt"), //
+                        hasNote("Transaktion-Nr.: 203036888"), //
+                        hasAmount("EUR", 382.12), hasGrossValue("EUR", 382.12), //
+                        hasTaxes("EUR", 0.00), hasFees("EUR", 0.00))));
+
+        // check security
+        assertThat(results, hasItem(security( //
+                        hasIsin("DE000SG1JXM7"), hasWkn(null), hasTicker(null), //
+                        hasName("SG EFF. INLINE09 DAX"), //
+                        hasCurrencyCode("EUR"))));
+
+        // check buy sell transaction
+        assertThat(results, hasItem(sale( //
+                        hasDate("2009-12-29"), hasShares(13.00), //
+                        hasSource("FinTechWertpapierAusgang07.txt"), //
+                        hasNote("Transaktion-Nr.: 203037029"), //
+                        hasAmount("EUR", 130.00), hasGrossValue("EUR", 130.00), //
+                        hasTaxes("EUR", 0.00), hasFees("EUR", 0.00))));
+    }
+
```

Comment:
We know that FlatEx has been renamed several times.

However, it is better that we assign the PDF debugs to the bank.
This document therefore still existed at the time of "biw AG".

Please rename file and test case to "testbiwAGWertpapierAusgang01" and move the test to the area (keep structure)

Reply:
ohh I missed that. So I renamed the Testcase and moved it and also the test file.